src/region/RegionCreator.tsx

```diff
@@ -154,18 +153,16 @@ export default function RegionCreator({ canDraw, className, onStart, onStop }: P
     const handleTouchStart = ({ targetTouches }: React.TouchEvent): void => {
         startDraw(targetTouches[0].clientX, targetTouches[0].clientY);
     };
-    const eventHandlers = canDraw
-        ? {
-              onClick: handleClick,
-              onMouseDown: handleMouseDown,
-              onMouseOut: handleMouseOut,
-              onMouseOver: handleMouseOver,
-              onTouchCancel: handleTouchCancel,
-              onTouchEnd: handleTouchEnd,
-              onTouchMove: handleTouchMove,
-              onTouchStart: handleTouchStart,
-          }
-        : {};
+    const eventHandlers = {
```

Comment:
Nice. We can just move these into the JSX now rather than having them as an object.